src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -1551,32 +1544,6 @@ impl<S: S3BackEnd + Send + Sync + 'static> S3MetaData<S> {
         result
     }
 
-    /// Invalidate cache from other nodes
-    async fn invalidate_remote(
-        &self,
-        full_ino: INum,
-        offset: i64,
-        len: usize,
-    ) -> DatenLordResult<()> {
-        dist_client::invalidate(
-            &self.kv_engine,
-            &self.node_id,
-            &self.volume_info,
-            full_ino,
-            offset
-                .overflow_div(self.data_cache.get_align().cast())
-                .cast(),
-            offset
-                .overflow_add(len.cast())
-                .overflow_sub(1)
-                .overflow_div(self.data_cache.get_align().cast())
-                .cast(),
-        )
-        .await
-        .map_err(DatenLordError::from)
-        .add_context("failed to invlidate others' cache")
-    }
```

Comment:
We should not remove `dist` related items in this PR, because the distributed cache is still working in this version! This will cause the CRON-CI test failed!

Reply:
I think what cause teh CRON-CCI test failed is the wrong config. I didn't replace all `--volume-info`.

Reply:
You can check the action's datenlord's log.

Reply:
I know, but they are not the same problem. I mean, it failed in my codes, when I try to remove the `dist` stuffs.

You can see <https://github.com/Reto911/datenlord/tree/rm-dist> for the minimized codes and <https://github.com/Reto911/datenlord/actions/runs/7210108246> for the failed test.

Reply:
I would prefer not to address the CRON-CI test issues in this PR

Reply:
Shall we disable it temporarily? Or the cron task will always fails when this PR is merged, which is annoying.